### ISSUE TITLE
Fix accessibility issues with Roadmaps

### ISSUE THREAD

**Issue by imperator99** [govau/dta-website-rebuild#668]
- [x] I’ve read and agree to the [Code of Conduct](./CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

# What is the context for your issue or request?

Given I am on the DTA website

# What action are you taking within that context?

When I visit a roadmap page

# What do you expect to happen when you take that action?

Then I should be able to to determine the roadmap item category through something other than colour.

**Comment by imperator99** [comment]
Resolved by https://github.com/govau/dta-website-rebuild/pull/674.
